app/models/customer.py

```diff
@@ -0,0 +1,32 @@
+from flask import current_app
+from sqlalchemy.orm import backref
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
```

Comment:
Instead of this being a column you can use the rentals to compute this.  

To do that it's helpful to add a relationship:
```suggestion
    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```